yarn.lock

```diff
@@ -14881,7 +14881,7 @@ reduce-component@1.0.1:
   resolved "https://registry.yarnpkg.com/reduce-component/-/reduce-component-1.0.1.tgz#e0c93542c574521bea13df0f9488ed82ab77c5da"
   integrity sha1-4Mk1QsV0UhvqE98PlIjtgqt3xdo=
 
-"reductio@github:crossfilter/reductio#0c45b03559":
+reductio@crossfilter/reductio#0c45b03559:
```

Comment:
I notice that this goes back and forth between these two formats pretty often and I'm really not sure why. It clearly doesn't make any functional difference, but it's a strange behavior that I'd eventually like to know the reason for.

Reply:
Yeah, I'm not sure why either.  I'm wondering if it's a yarn version thing.